go.mod

```diff
@@ -5,16 +5,16 @@ go 1.19
 require (
 	github.com/ghodss/yaml v1.0.0
 	github.com/hashicorp/go-version v1.6.0
-	github.com/hashicorp/terraform-plugin-sdk/v2 v2.27.0
-	github.com/rancher/rke v1.4.6
+	github.com/hashicorp/terraform-plugin-sdk/v2 v2.29.0
+	github.com/rancher/rke v1.4.10
 	github.com/satori/go.uuid v1.2.1-0.20181028125025-b2ce2384e17b
-	github.com/sirupsen/logrus v1.8.1
+	github.com/sirupsen/logrus v1.9.3
 	gopkg.in/yaml.v2 v2.4.0
-	k8s.io/api v0.25.4
-	k8s.io/apimachinery v0.25.4
-	k8s.io/apiserver v0.25.4
-	k8s.io/client-go v0.25.4
-	k8s.io/kubernetes v1.13.0
+	k8s.io/api v0.27.6
```

Comment:
Can we change to `v0.25.12` to match what is pinned in the [go.mod](https://github.com/rancher/rke/blob/v1.4.10/go.mod)  file in rke 1.4.10?